packages/app/features/activity/screen.tsx

```diff
@@ -180,7 +182,9 @@ function Suggestions() {
                 <Spinner size="small" color="$send1" />
               </Avatar.Fallback>
             </Avatar>
-            <Paragraph>@{user.username}</Paragraph>
+            <Paragraph color="$white" fontFamily="$mono">
```

Comment:
```suggestion
            <Paragraph $theme-dark={{color: "$white"}} fontFamily="$mono">
```
and then do the appropriate `$theme-light` for light version.